V2/time_tracker/time_entries/_application/_time_entries/_delete_time_entry.py

```diff
@@ -0,0 +1,38 @@
+import json
+
+import azure.functions as func
+
+from ... import _domain
+from ... import _infrastructure
+from time_tracker._infrastructure import DB
+
+DATABASE = DB()
+
+
+def delete_time_entry(req: func.HttpRequest) -> func.HttpResponse:
+    time_entry_dao = _infrastructure.TimeEntriesSQLDao(DATABASE)
+    time_entry_service = _domain.TimeEntryService(time_entry_dao)
+    use_case = _domain._use_cases.DeleteTimeEntryUseCase(time_entry_service)
+
+    try:
+        time_entry_id = int(req.route_params.get("id"))
```

Comment:
we need to validate if the time entry exists first

Reply:
I don't know if it's necessary, because delete function does it internally